.github/workflows/reusable-build-flavor.yaml

```diff
@@ -35,199 +35,45 @@ jobs:
-          path: |
-            grype-results/*.sarif
-            trivy-results/*.sarif
-          if-no-files-found: error
+          mkdir build
+          docker run --rm -v /var/run/docker.sock:/var/run/docker.sock \
+          -v $PWD:/output \
+          quay.io/kairos/auroraboot:latest --debug build-iso --output /output/ \
+          --name kairos-${{ inputs.flavor }}-${{ inputs.flavor_release }}-${{ inputs.variant }}-${{ inputs.arch }}-${{ env.GIT_VERSION }} \
+          docker:${{ inputs.flavor }}-init:${{ inputs.flavor_release }}
       - uses: actions/upload-artifact@65c4c4a1ddee5b72f698fdd19549f0f0fb45cf08 # v4
         with:
-          name: kairos-${{ inputs.flavor }}-${{ inputs.flavor_release }}.iso.zip
+          name: kairos-${{ inputs.flavor }}-${{ inputs.flavor_release }}-${{ inputs.variant }}.iso.zip
           path: |
             *.iso
             *.sha256
             versions.yaml
           if-no-files-found: error
-      - uses: actions/upload-artifact@65c4c4a1ddee5b72f698fdd19549f0f0fb45cf08 # v4
-        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
-        with:
-          name: kairos-${{ inputs.flavor }}-${{ inputs.flavor_release }}.sbom.zip
-          path: |
-            *.syft.json
-            *.spdx.json
-          if-no-files-found: error
-      - name: Push to quay
-        if: ${{ github.event_name == 'push' && github.ref == 'refs/heads/master' }}
-        env:
-          COSIGN_YES: true
-        run: |
-          IMAGE=$(FLAVOR=${{ inputs.flavor }} FLAVOR_RELEASE="${{ inputs.flavor_release }}" MODEL=${{ inputs.model }} TARGETARCH=${{ inputs.arch }} VARIANT=${{ inputs.variant}} REGISTRY_AND_ORG="quay.io/kairos" RELEASE=master kairos-agent versioneer container-artifact-name)
-          docker tag $(cat IMAGE) $IMAGE
-          docker push $IMAGE
-          image_ref=$(docker image inspect --format='{{index .RepoDigests 0}}' "$IMAGE")
-          # TODO: on master we don't want to push all the sbom files since the tag is always the same and we don't go back to clean the old sbom files
-          # but we should also use this reusable job for releases and then it's important to do it
-          # For master:
-          cosign sign $image_ref
-          # For releases:
-          # spdx=$(ls *.spdx.json)
-          # cosign attach sbom --sbom $spdx $image_ref
-          # cosign sign $image_ref --attachment sbom
-          # in-toto attestation
-          # cosign attest --type spdx --predicate $spdx $image_ref
       - name: Push to testing
         run: |
-          _IMG=$(cat IMAGE)
-          TEST_IMAGE="ttl.sh/kairos-${{ inputs.flavor }}-${{ inputs.flavor_release }}-${{ github.sha }}:24h"
-          docker tag $_IMG $TEST_IMAGE
+          TEST_IMAGE="ttl.sh/kairos-${{ inputs.flavor }}-${{ inputs.flavor_release }}-${{ inputs.variant }}-${{ github.sha }}:24h"
+          docker tag ${{ inputs.flavor }}-init:${{ inputs.flavor_release }} $TEST_IMAGE
```

Comment:
I think the same is used for the `uki` test, so I think it might be problematic if the two are running at the same time, no?

Reply:
Umm good point on the naming, indeed. I need to check this further but the idea would be to push the same image for both arches so they should share the name. Unfortunately I less you build it at the same time, they step over each other so not there yet. Plus this affects only PR and master I think, release still do its thing.

Also I think that we don't push the images or at least that's the idea, everything local instead of wasting time pushing and pulling unless it's necessary 

I will have a look at this stuff

Reply:
done and done, now it has arches in them for the normal jobs in which we need to push them to be consumed. For uki, we dont push those init images so its not necessary